File: example-scripts/hello.py
#!/usr/bin/env python3

import sys
from datetime import datetime

print("🎉 Hello from Script Launcher!")
print("=" * 32)
print("Cet script Python a été exécuté avec succès.")
print()
print("Date et heure actuelle:")
print(datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
print()
print("Version Python:")
print(sys.version)
print()
print("✅ Script terminé!")

File: example-scripts/hello.js
#!/usr/bin/env node

console.log("🎉 Hello from Script Launcher!");
console.log("================================");
console.log("Cet script Node.js a été exécuté avec succès.");
console.log();
console.log("Date et heure actuelle:");
console.log(new Date().toLocaleString());
console.log();
console.log("Version Node.js:");
console.log(process.version);
console.log();
console.log("Arguments reçus:");
console.log(process.argv.slice(2));
console.log();
console.log("✅ Script terminé!");
